test/verify/check-machines

```diff
@@ -516,5 +528,436 @@ class TestMachines(MachineCase):
 
         self.assertNotIn(name, m.execute("virsh list --all"))
 
+    def testCreate(self):
+        """
+        this test will print many expected error messages
+        """
+
+        b = self.browser
+        m = self.machine
+        s = self
+
+        runner = TestMachines.CreateVmRunner(b, m)
+
+        self.startLibvirt()
+
+        self.login_and_go("/machines")
+        b.wait_in_text("body", "Virtual Machines")
+
+        def checkEnvIsEmpty():
+            def checkEnvIsEmptyImpl(sleep=1):
+                # wait for the vm and disks to be deleted
+                time.sleep(sleep)
```

Comment:
Can we use of b.wait(lambda) to be consistent with the rest of the code?
I think it's more stable to wait for outcome instead of experimentally found delay.